umap/static/umap/js/umap.core.js

```diff
@@ -428,6 +441,7 @@ L.U.Help = L.Class.extend({
     shortCredit: L._('Will be displayed in the bottom right corner of the map'),
     longCredit: L._('Will be visible in the caption of the map'),
     sortKey: L._('Property to use for sorting features'),
+    slugKey: L._('The name of the property to use as feature unique identifier.'),
```

Comment:
Oh. This is a new string to translate :)
I don't kwnow yet how tu update the english "source" file on Transifex.

Reply:
`make tx_push` ;)

But I usually don't do it until the feature branch is merged on master.

Reply:
Ah, sorry:

```
# Collect the new strings
make messages
# Push to transifex
make tx_push
```